src/nvim/testdir/test_stat.vim

```diff
@@ -1,47 +1,54 @@
 " Tests for stat functions and checktime
 
 func Test_existent_file()
-  let fname='Xtest.tmp'
+  let fname = 'Xtest.tmp'
 
-  let ts=localtime()
-  sleep 1
-  let fl=['Hello World!']
+  let ts = localtime()
+  let fl = ['Hello World!']
   call writefile(fl, fname)
-  let tf=getftime(fname)
-  sleep 1
-  let te=localtime()
+  let tf = getftime(fname)
+  let te = localtime()
 
   call assert_true(ts <= tf && tf <= te)
   call assert_equal(strlen(fl[0] . "\n"), getfsize(fname))
   call assert_equal('file', getftype(fname))
   call assert_equal('rw-', getfperm(fname)[0:2])
+
+  call delete(fname)
 endfunc
 
 func Test_existent_directory()
-  let dname='.'
+  let dname = '.'
 
   call assert_equal(0, getfsize(dname))
   call assert_equal('dir', getftype(dname))
   call assert_equal('rwx', getfperm(dname)[0:2])
 endfunc
 
 func Test_checktime()
-  let fname='Xtest.tmp'
+  let fname = 'Xtest.tmp'
 
-  let fl=['Hello World!']
+  let fl = ['Hello World!']
   call writefile(fl, fname)
   set autoread
   exec 'e' fname
-  sleep 2
-  let fl=readfile(fname)
+  " FAT has a granularity of 2 seconds, otherwise it's usually 1 second
+  if has('win32')
+    sleep 2
+  else
+    sleep 1
+  endif
```

Comment:
Unlike Vim, we run ASAN on travis. It's too slow, so we need to keep this at 2 seconds.

```
Found errors in Test_checktime():
function RunTheTest[13]..Test_checktime line 17: Expected 'Hello World! - checktime' but got 'Hello World!'

Test results:


From test_stat.vim:
Found errors in Test_checktime():
function RunTheTest[13]..Test_checktime line 17: Expected 'Hello World! - checktime' but got 'Hello World!'
TEST FAILURE
```